backend/src/packages/users/user.repository.ts

```diff
@@ -37,6 +37,17 @@ class UserRepository implements IRepository {
     return result[0] ? UserEntity.initialize(result[0]) : null;
   }
 
+  public async findByEmail(value: string): Promise<UserEntity | null> {
+    const result = await this.db
+      .driver()
+      .select()
+      .from(this.usersSchema)
+      .where(eq(this.usersSchema.email, value))
```

Comment:
Please verify that this equality check is not case-sensitive
The next case should return true: 
db value: someuser@gmail.com
incoming value: someUser@gmail.com